torchrl/__init__.py

```diff
@@ -88,3 +91,8 @@ def _check_for_faulty_process(processes):
         raise RuntimeError(
             "At least on process failed. Check for more infos in the log."
         )
+def seed_generator(seed, repeat):
+    MAX_SEED_VAL =  2**32 - 1
+    rng = np.random.default_rng(seed)
+    for i in range(repeat):
+        yield int.from_bytes(rng.bytes(8), "big") % MAX_SEED_VAL
```

Comment:
There should be a new line (linter)
Refer to CONTRIBUTING.md on the main branch: you can use `pre-commit` which will run the lint for you and make it all work!